stories/carousel/carousel.stories.js

```diff
@@ -0,0 +1,480 @@
+                aria-label="Go to next item">
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-6" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+
+    <h4>Numeral Format for Page Indicator</h4>
+    <div 
+        class="fd-carousel" 
+        data-ride="carousel" 
+        style="margin-bottom: 3rem; max-width: 30rem;">
+        <div class="fd-carousel__page-indicator-container">
+            <div class="fd-carousel__page-indicators">
+                <div class="fd-carousel__text">1 of 4</div>
+            </div>
+        </div>
+        <div class="fd-carousel__content" style="min-height: 15.5rem;">
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-left"
+                data-slide="prev"
+                aria-label="Go to previous item">
+            </button>
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next"
+                aria-label="Go to next item">
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-7" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+
+    <h4>No Page Indicators</h4>
+    <div 
+        class="fd-carousel" 
+        data-ride="carousel" 
+        style="margin-bottom: 3rem; max-width: 30rem;">
+        <div class="fd-carousel__page-indicator-container">
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-left"
+                data-slide="prev"
+                aria-label="Go to previous item">
+            </button>
+            <ol class="fd-carousel__page-indicators"></ol>
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next"
+                aria-label="Go to next item">
+            </button>
+        </div>
+        <div class="fd-carousel__content" style="text-align: center; padding: 1rem; min-height: 15.5rem;">
+            <div class="fd-carousel__item">
+                <img src="http://lorempixel.com/400/400/nature/1/" alt="Carousel Image 1" />
+            </div>
+            <div class="fd-carousel__item">
+                <img src="http://lorempixel.com/400/400/nature/2/" alt="Carousel Image 2" />
+            </div>
+            <div class="fd-carousel__item">
+                <img src="http://lorempixel.com/400/400/nature/3/" alt="Carousel Image 3" />
+            </div>
+            <div class="fd-carousel__item fd-carousel__item--active">
+                <img src="http://lorempixel.com/400/400/nature/4/" alt="Carousel Image 4" />
+            </div>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-8" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+</div>
+`;
+
+carouselTop.storyName = 'Carousel with Page Indicator on Top';
```

Comment:
"The top page indicator carousel displays a content area, navigation buttons and a page indicator at the top."